packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -32,7 +33,7 @@ describe('SpinButton', () => {
     const exampleDefaultValue: string = '12';
 
     const renderedDOM: HTMLElement = renderIntoDocument(
-      <SpinButton
+      <SpinButtonBase
```

Comment:
I had to switch the tested components to the SpinButtonBase. Using the styled component, renderIntoDocument() returns null.

Reply:
The snapshot test uses the styled SpinButton